webpack.config.js

```diff
@@ -108,7 +108,12 @@ function webpackConfig(env = {}) {
     },
     output: {
       publicPath: "/",
-      filename: "assets/[name].[contenthash].js",
+      filename: chunkData => {
```

Comment:
* [ ] Q: How about an `import '!!file-loader!...'` in the component? Moves the complexity to where it is used.